.travis.yml

```diff
@@ -28,6 +28,14 @@ matrix:
       after_success:
         - export COVR_RUNNING=true
         - Rscript -e 'covr::codecov()'
+    - language: java
+      dist: trusty
+      cache: packages
+      before_install:
+        - export MLFLOW_TEST_REQUIREMENTS=false
```

Comment:
What this

Reply:
Lol good question, tried to make this logic clearer - basically we want a way to not install Python dev/test dependencies in non-Python tests, we were using this env variable before but I switched to checking Travis's TRAVIS_PYTHON_VERSION env var, which is only set if we're running Python tests ([see docs](https://docs.travis-ci.com/user/environment-variables/))